images/manageiq-base/Dockerfile

```
@@ -45,7 +45,7 @@ COPY container-assets/clean_dnf_rpm /usr/local/bin/
 
 RUN curl -L https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo > /etc/yum.repos.d/ansible-runner.repo
 
-RUN dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install \
+RUN dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs --setopt=install_weak_deps=False install \
```

```suggestion
RUN dnf config-manager --setopt=tsflags=nodocs --setopt=install_weak_deps=False --save && \
    dnf -y --disableplugin=subscription-manager install \
```
Then we can drop those settings from all following commands.

---

Not sure if we want to put subscription-manager in there too

---

I was gonna ask if you wanted to do that - do we want that for this PR or a followup ?

---

> Not sure if we want to put subscription-manager in there too

Unfortunately, we still need subscription-manager for the s390x builds.

---

Ok, https://github.com/ManageIQ/manageiq-pods/pull/847